documentation/swagger-checklist.md

```diff
@@ -84,6 +84,13 @@ The Azure Developer Experience team will continue to deliver rich tools to valid
 
 - [ ] **M1000**: Plural names in parameters and model properties MUST only be used to represent collections.
 
+  Example from [arm-cdn](https://github.com/Azure/azure-rest-api-specs/blob/master/arm-cdn/2016-10-02/swagger/cdn.json):
+  ```json
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Cdn/profiles": {
```

Comment:
@RobDolinMS : Example here does not point to plural names. Am i confused here? Could you please point me to the parameters/model properties which are collections?